common/config/azure-pipelines/templates/publish-tarballs.yaml

```diff
@@ -0,0 +1,8 @@
+parameters:
+  - name: Publish
+    type: string
+    default: ''
+
+steps:
+  - script: 'node common/scripts/install-run-rush.js publish $(Publish) --pack --include-all --release-folder esrp-release'
```

Comment:
The folder specified under `--release-folder` should correspond to an `output` entry in `templateContext.outputs`.